CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 2.14.0 (2018-11-29)
+- Added `stache-hide-from-search` component to allow users to remove content from Stache Search results. [#557](https://github.com/blackbaud/stache2/pull/557)
+- Fixed alignment issue with sidebar and table of contents alignment. [#556](https://github.com/blackbaud/stache2/pull/556)
+- Updated version of `skyux-builder`.
+- Updated Travis build options to enhance test utility.
+- Updated skyux dependencies to depend on components directly. [#553](https://github.com/blackbaud/stache2/pull/553)
+- Updated to allow `Action Button` comopnent to be used without icons.
+- Updated `Action Button` component to be accessible. [#548](https://github.com/blackbaud/stache2/pull/548)
+- Update `Code Block` component to use accessible code highlighting coloring. [#551](https://github.com/blackbaud/stache2/pull/551)
```

Comment:
"Updated" should be past tense like the other items. "Code Block" should not be capitalized; it should be the element name, which should be "stache-code-block" here. Also, "accessible code highlighting coloring" is a mouthful and hard to follow. Based on the PR description, I'd change this entry to "Updated `stache-code-block` component to improve accessibility." If you think we need to say more about the specific accessibility change, I'd "Updated `stache-code-block` component to improve accessibility by increasing the color contrast."